src/analyses/constant_propagator.cpp

```diff
@@ -93,7 +93,7 @@ void constant_propagator_domaint::transform(
   {
     exprt g;
 
-    if(from->get_target()==to)
+    if(from->get_target()==to)  // Safe as GOTO implies to and from comparable
```

Comment:
As those comments may later be read out of context: how about "comparing iterators is safe as the target must be within the same list of instructions"?

Reply:
Done.